content/spaces/observability.md

```diff
@@ -99,7 +99,7 @@ spec:
     port: metrics
     scheme: http
     scrapeTimeout: 10s
-  - relabelings:
+    relabelings:
```

Comment:
That is a subtle fix. :)

Reply:
Definitely one of the more mischievous hyphens I've encountered.